Containerfile

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright (C) 2025 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Ubuntu 24.04
+FROM node:22-bookworm-slim
```

Comment:
use Red Hat UBI image for node, avoid Dockerhub

Reply:
Check next comment,  need debian based and I want something official, so I could find quay images either so need to rely in Dockerhub.

Also for building is fine (we will not hit rate limits or something) as long as we do not publish there... and for it I would expect images published on quay and ghcr

Reply:
And this is not for a Red Hat product, this is upstream open source project so it's fine.

Reply:
Ok, not a Red Hat product ... but will you use Red Hat infrastructure to pull these images ? if so, then it's a no-go